network/src/commonMain/kotlin/com/wire/kalium/network/utils/NetworkResponse.kt

```diff
@@ -43,3 +42,13 @@ fun <T : Any> NetworkResponse<T>.isSuccessful(): Boolean {
     }
     return this@isSuccessful is NetworkResponse.Success
 }
+
+suspend fun <T : Any, R : Any> NetworkResponse<T>.flatMap(
```

Comment:
```suggestion
internal suspend fun <T : Any, R : Any> NetworkResponse<T>.flatMap(
```
Functions with generics must be internal to avoid accidental use on iOS